docs/ldml/tr35-keyboards.md

```diff
@@ -520,207 +595,179 @@ Indicates that:
-
-```xml
-<keyMap modifiers="ctrl+shift?">
-    …
-</keyMap>
-```
-
-```xml
-<keyMap modifiers="ctrl">
-    …
-</keyMap>
-```
-
-Modifier Examples:
-
-```xml
-<keyMap modifiers="cmd?+opt+caps?+shift" />
-```
-
-Caps-Lock may be ON or OFF, Option must be ON, Shift must be ON and Command may be ON or OFF.
-
-```xml
-<keyMap modifiers="shift caps" />
-```
-
-Caps-Lock must be ON OR Shift must be ON.
-
-If the `modifiers` attribute is not present on a `keyMap` then that particular key map is the base map.
 
 * * *
 
-### 5.9 <a name="Element_map" href="#Element_map">Element: map</a>
-
-This element defines a mapping between the base character and the output for a particular set of active modifier keys. This element must have the `keyMap` element as its parent.
+### 5.9 <a name="Element_key" href="#Element_key">Element: key</a>
 
-If a `map` element for a particular ISO layout position has not been defined then if this key is pressed, no output is produced.
+This element defines a mapping between an abstract key and its output. This element must have the `keys` element as its parent. The `key` element is referened by the `keys=` attribute of the [`row` element](#Element_row).
 
 **Syntax**
 
 ```xml
-<map
- iso="{the iso position}"
- to="{the output}"
+<key
+ id="{key id}"
+ [flicks="{flicks identifier}"]
+ [gap="true"]
  [longPress="{long press keys}"]
- [transform="no"]
+ [longPressDefault="{default longpress target}"]
  [multitap="{the output on subsequent taps}"]
- [longPress-status="optional"]
- [optional="{optional mappings}"]
- [hint="{hint to long press content}"]
+ [switch="{layer id}"]
+ [to="{the output}"]
+ [transform="no"]
+ [width="{key width}"]
  /><!-- {Comment to improve readability (if needed)} -->
 ```
 
 > <small>
 >
-> Parents: [keyMap](#Element_keyMap)
+> Parents: [keys](#Element_keys)
 > Children: _none_
-> Occurence: optional, multiple
->
+> Occurrence: optional, multiple
 > </small>
 
-_Attribute:_ `iso` (exactly one of base and iso is required)
+**Note**: The `id` attribute is required, as is _at least one of_ `switch`, `gap`, or `to`.
 
-> The `iso` attribute represents the ISO layout position of the key (see the definition at the beginning of the document for more information).
+_Attribute:_ `id`
 
-_Attribute:_ `to` (required)
+> The `id` attribute uniquely identifies the key. NMTOKEN, restricted to "[a-zA-Z0-9_.-]". It can (but needn't be) the Latin key name for a Latn script keyboard (a, b, c, A, B, C, …), or any other valid token (e-acute, alef, alif, alpha, …)
+
+_Attribute:_ `flicks="flick-id"` (optional)
+
+> The `flicks` attribute indicates that this key makes use of a [`flicks`](#Element_flicks) set with the specified id.
+
+_Attribute:_ `gap="true"` (optional)
+
+> The `gap` attribute indicates that this key does not have any appearance, but represents a "gap" of the specified number of key widths. Can be used with `width` to set a width.
```

Comment:
Same as prior comment:  contrast with [en-t-k0-teletype.xml](https://github.com/unicode-org/cldr/pull/1847/files#diff-9b1c99aec1ff6f44a068a703a57c0a534183941f44db9128771fff610cc9d0e1), ensure consistency.

Reply:
Gap=true is what we discussed in committee.